src/04-modality-specific-files/01-magnetic-resonance-imaging-data.md

```diff
@@ -55,21 +55,27 @@ that a given scan was collected with the intended coil elements selected
+| MTNumberOfPulses            | RECOMMENDED if the MTstate is `True`.          | [number][]                             | The number of magnetization transfer RF pulses applied before the readout.                                                                                                                                                                                         |
 | MTPulseShape                | RECOMMENDED if the MTstate is `True`.          | [string][]                             | Shape of the magnetization transfer RF pulse waveform. Accepted values: `HARD`, `GAUSSIAN`, `GAUSSHANN` (gaussian pulse with Hanning window), `SINC`, `SINCHANN` (sinc pulse with Hanning window), `SINCGAUSS` (sinc pulse with Gaussian window), `FERMI`.                |
-| MTPulseDuration             | RECOMMENDED if the MTstate is `True`.          | [number][]                             | Duration of the magnetization transfer RF pulse in seconds.                                                                                                                                                                                                               |
+| MTPulseDuration             | RECOMMENDED if the MTstate is `True`.          | [number][]                             | Duration of the magnetization transfer RF pulse in seconds.                                                                                                                                                                                          |
+| SpoilingState               | RECOMMENDED                                    | [boolean][]                            | Boolean stating whether the pulse sequence uses any type of spoiling strategy to suppress residual transverse magnetization.                                                                                                                                           |
+| SpoilingType                | RECOMMENDED if the SpoilingState is `True`.    | [string][]                             | Specifies which spoiling method(s) are used by a spoiled sequence. Accepted values: `RF`, `GRADIENT` or `COMBINED`.                                                                                                                                                  |
+| SpoilingRFPhaseIncrement    | RECOMMENDED if the SpoilingType has `RF`.      | [number][]                             | The amount of incrementation described in degrees, which is applied to the phase of the excitation pulse at each TR period for achieving RF spoiling.                                                                                                                        |
```

Comment:
> RECOMMENDED if the SpoilingType has `RF`.

The type for `SpoilingType` is "string", but saying "has" here implies (to me at least) that it could be a list. I.e., `SpoilingRFPhaseIncrement` is recommended if the field `SpoilingType` _includes_ `RF`, rather than _is_ `RF`.

TL;DR: Should this be "is" instead?

Reply:
37db200 should resolve this confusion.